configurations/derived_products/item_10.yaml

```diff
@@ -0,0 +1,147 @@
+## Define inputs to perform statistics on
+global_attributes:
+  title: ITEM 
+sources:
+  - product: ls5_nbar_albers
+    name: ITEM 
+    measurements: [blue, green, red, nir, swir1, swir2]
+    group_by: solar_day
+#    source_filter:
+#      product: ls5_level1_scene
+#      gqa_cep90: (-0.25, 0.25)
+    masks:
+      - product: ls5_pq_albers
+        measurement: pixelquality
+        group_by: solar_day
+        fuse_func: datacube.helpers.ga_pq_fuser
+        flags:
+          contiguous: True
+          cloud_acca: no_cloud
+          cloud_fmask: no_cloud
+          cloud_shadow_acca: no_cloud_shadow
+          cloud_shadow_fmask: no_cloud_shadow
+          blue_saturated: False
+          green_saturated: False
+          red_saturated: False
+          nir_saturated: False
+          swir1_saturated: False
+          swir2_saturated: False
+
+  - product: ls7_nbar_albers
+    name: ITEM
+    measurements: [blue, green, red, nir, swir1, swir2]
+    group_by: solar_day
+#    source_filter:
+#      product: ls7_level1_scene
+#      gqa_cep90: (-0.25, 0.25)
+    masks:
+      - product: ls7_pq_albers
+        measurement: pixelquality
+        group_by: solar_day
+        fuse_func: datacube.helpers.ga_pq_fuser
+        flags:
+          contiguous: True
+          cloud_acca: no_cloud
+          cloud_fmask: no_cloud
+          cloud_shadow_acca: no_cloud_shadow
+          cloud_shadow_fmask: no_cloud_shadow
+          blue_saturated: False
+          green_saturated: False
+          red_saturated: False
+          nir_saturated: False
+          swir1_saturated: False
+          swir2_saturated: False
+
+
+  - product: ls8_nbar_albers
+    name: ITEM
+    measurements: [blue, green, red, nir, swir1, swir2]
+    group_by: solar_day
+#    source_filter:
+#      product: ls8_level1_scene
+#      gqa_cep90: (-0.25, 0.25)
+    masks:
+      - product: ls8_pq_albers
+        measurement: pixelquality
+        group_by: solar_day
+        fuse_func: datacube.helpers.ga_pq_fuser
+        flags:
+          contiguous: True
+          cloud_acca: no_cloud
+          cloud_fmask: no_cloud
+          cloud_shadow_acca: no_cloud_shadow
+          cloud_shadow_fmask: no_cloud_shadow
+          blue_saturated: False
+          green_saturated: False
+          red_saturated: False
+          nir_saturated: False
+          swir1_saturated: False
+          swir2_saturated: False
+
+
+## Define whether and how to chunk over time
+date_ranges:
+  start_date: 2013-01-01
+  end_date: 2015-01-01
+  stats_duration: 2y
```

Comment:
It's no longer necessary to specify `stats_duration`, `step_size` or `type` unless necessary. ie. When performing a stat that isn't over the entire start-end date range.